README.md

```diff
@@ -92,6 +92,12 @@ Then, run the monitoring deployment playbook:
 
     ansible-playbook --vault-password-file vault-password -i ansible/inventory -e @config/swarm-sip.yml ansible/monitoring-monasca-container.yml
 
+# Deploying and managing Kubernetes
+
+The deployment instructions for Kubernetes is identical to Docker Swarm. There is an additional playbook available to handle upgrades which can be invoked as follows:
+
+    ansible-playbook --vault-password-file vault-password -i ansible/inventory-k8s -e version=1.11.2 ansible/container-infra-upgrade.yml
```

Comment:
Not being well acquainted with this I am left wondering what ```version``` refers to. Could it be called something more specific or set as a default?

Reply:
Thanks for pointing this out, i've clarified it as k8s_version and also added a default variable.